addons/sale/models/product_template.py

```diff
@@ -266,6 +268,7 @@ def _get_combination_info(self, combination=False, product_id=False, add_qty=1,
             'price': price,
             'list_price': list_price,
             'has_discounted_price': has_discounted_price,
+            'price_per_uom': price / uom_count,
```

Comment:
uom_count = product_template.uom_count... since it is a compute with the falllback -> product.product -> produt.template -> 1 already !